packages/experiments/src/components/Stack/Stack.tsx

```diff
@@ -52,11 +52,11 @@ const view: IStackComponent['view'] = props => {
 
   if (wrap) {
     return (
-      <div className={classNames.root}>
-        <RootType {...nativeProps} className={classNames.inner}>
```

Comment:
@kkjeer is there any reason why `RootType` is a child of `div` instead of the other way around? I changed it because it conforms better with the slots approach and didn't find any differences but wanted to ask your take on the matter.

Reply:
This change makes sense to me.